build.gradle

```diff
@@ -109,7 +110,10 @@ configure(javaSubprojects) {
         testCompile 'org.hamcrest:hamcrest-junit:2.0.0.0'
 
         testCompile 'org.mockito:mockito-core:1.10.19'
+        testCompile 'org.powermock:powermock-mockito-release-full:1.6.2'
+	testCompile 'ObjectFaker:ObjectFaker:0.1-SNAPSHOT'
```

Comment:
Would be better to use only release version, especially since we are releasing soon.